CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [BUGFIX] Fix `container_memory_usage_bytes:sum` recording rule #1865
 * [BUGFIX] Fix `MimirGossipMembersMismatch` alerts if Mimir alertmanager is activated #1870
 * [BUGFIX] Fix `MimirRulerMissedEvaluations` to show % of missed alerts as a value between 0 and 100 instead of 0 and 1. #1895
+* [BUGFIX] Fix `MimirCompactorHasNotUploadedBlocks` alert false positive when Mimir is deployed in monolithic mode. #1901
```

Comment:
```suggestion
* [BUGFIX] Fix `MimirCompactorHasNotUploadedBlocks` and `MimirCompactorSkippedBlocksWithOutOfOrderChunks` alerts false positive when Mimir is deployed in monolithic mode. #1901
```

Reply:
See PR description:
> For consistency I've done the same change to MimirCompactorSkippedBlocksWithOutOfOrderChunks alert too, even if this alert doesn't suffer the same issue because the metric it queries is just exported by the compactor.

I changed for consistency, but wasn't suffering any false positive, reason why it's not mentioned in the CHANGELOG.